ic-agent/src/agent/route_provider/dynamic_routing/dynamic_route_provider.rs

```diff
@@ -194,7 +200,7 @@ where
     ///   - Starts/stops health check tasks (HealthCheckActors) based on the newly added/removed nodes.
     ///   - These spawned health check tasks periodically update the snapshot with the latest node health info.
     pub async fn run(&self) {
-        info!("{DYNAMIC_ROUTE_PROVIDER}: started ...");
+        log!(info, "{DYNAMIC_ROUTE_PROVIDER}: started ...");
```

Comment:
How to instantiate a tracing subscriber, which captures these messages? My previous approach of using [tracing_subscriber](https://crates.io/crates/tracing-subscriber) doesn't seem to work anymore with this `log!()`.

Reply:
The purpose of the log! macro is to do nothing if the `tracing` feature is not enabled, to avoid polluting the log sites with cfgs. If tracing-subscriber did nothing you likely do not have that feature enabled.

Reply:
Gotcha, yes when setting `--features tracing` i do see the messages.